airflow/providers/apache/hdfs/hooks/webhdfs.py

```diff
@@ -153,3 +153,16 @@ def load_file(
             hdfs_path=destination, local_path=source, overwrite=overwrite, n_threads=parallelism, **kwargs
         )
         self.log.debug("Uploaded file %s to %s", source, destination)
+
+    def read_file(self, filename: str) -> str:
+        """
+        Reads a file from HDFS
```

Comment:
```suggestion
        """Reads a file from HDFS.
```

We’re working to comply to this docstring format. The migration is in progress but this one would break linters when the check is enforced.